tsconfig.base.json

```diff
@@ -65,9 +65,6 @@
       "@o3r/analytics": [
         "packages/@o3r/analytics/src/public_api"
       ],
-      "@o3r/analytics/*": [
```

Comment:
Needed for the `plugins` subentrie, can be replaced by `@o3r/analytics/plugins/*` if it is fixing your issue